elements/pfe-primary-detail/src/pfe-primary-detail.scss

```diff
@@ -81,23 +125,123 @@ $LOCAL-VARIABLES: (
+      // Make the header as wide as the full column
+      width: calc(100% + #{$horizontal-padding * 2});
+      // Make sure it's flush with the left edge
+      margin-left: ($horizontal-padding * -1);
+    }
+  }
+}
+
+#details-wrapper__header {
+  display: none;
+  grid-row: 1 / span 1;
+  grid-column: 1 / span 1;
+  align-self: start;
+  justify-self: start;
+
+  :host([breakpoint='compact']) & {
+    display: flex;
+    align-items: flex-start;
+  }
+}
+
+#details-wrapper__back {
+  display: none;
+  position: relative;
+  align-self: stretch;
+  // min-width: 32px;
+  // min-height: 40px;
+  border: 0;
+  // overflow: hidden;
+  background: transparent;
+  // text-indent: -9999em; // Hiding assistive text that explains the back icon
+  font-size: inherit;
+  font-family: inherit;
+  font-weight: bold;
+  cursor: pointer;
+  text-align: left;
+
+  // @todo: (KS) figure out how to get focus outline on back button - psuedo :focus is not working here
+  :host([active]) & {
+    display: inline-block;
+    width: 100%;
+    padding-left: 16px;
+  }
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 0%;
+    display: block;
+    width: 0.75em;
+    height: 0.75em;
+    box-shadow: inset -0.1875em -0.1875em 0 0.0625em #000;
+    transform: translate(-50%, -50%) scale(0.6) rotate(135deg);
+  }
+}
+
+#details-wrapper__heading {
+  margin: 0;
+  padding: 0;
+  font-size: inherit;
+  line-height: 1.5;
+  // Add top padding that's half of mininmum back button size
+  padding: 0.5em 0;
 }
 
 ::slotted([slot="details"]) {
-  display: none!important;
 
   :host([consistent-height]) & {
     display: block!important;
     grid-row: 1 / span 1!important;
     grid-column: 1 / span 1!important;
-    visibility: hidden!important;
+  }
+
+  :host([consistent-height][breakpoint='compact']) & {
+    grid-row: 2 / span 1!important;
   }
 }
 
-::slotted([aria-hidden='false']) {
-  display: block!important;
+::slotted([hidden]) {
+  visibility: hidden!important;
+  // Using position absolute so grid doesn't account for them in the layout
+  position: absolute!important;
 
   :host([consistent-height]) & {
-    visibility: visible!important;
+    // Consistent height is setup so we want unopen panes to have a height that grid accounts for
+    position: static!important;
   }
 }
+
+::slotted([slot="details-nav--footer"]) {
+  border-right: pfe-local(Border) !important;
+}
+
+// @todo: add note in readme with focus indicator example and show ppl that they need to add a good focus indicator to focusable elements
+// @todo: (KS) figure out if we still need this now that we have the lightdom stylesheet
+::slotted(.is-focused:focus),
+::slotted(.is-focused:hover) {
+  outline: 1px dashed #000 !important;
+  outline-width: 2px !important;
+}
+
+::slotted(ul.is-focused:hover) {
+  outline: 0 !important;
+}
+
+::slotted(a.is-focused) {
+  // Increase clickable area of links
+  padding: 8px !important;
```

Comment:
This won't work unless we make the links if they aren't `inline` and it will cause a reflow of text, which isn't ideal.

I think we have to leave hit area size to the authors, something like this is likely to cause bugs, IMO.

Reply:
Hey @wesruv that is also old stuff that can be removed.